roles/test/behaviour.role

```diff
@@ -0,0 +1,53 @@
+nuclear_role(
```

Comment:
how come this is in this PR?

Reply:
was needed to test path planning and wasn't to added when Tester module was created